packaging/fedora/snapd.spec

```diff
@@ -512,7 +516,7 @@ rm -rf vendor/*
 %build
 
 # Build snapd
-mkdir -p src/github.com/snapcore
+mkdir -p src/github.com/snapcore && rm -f src/github.com/snapcore/snapd
```

Comment:
Why is this neccessary?

Reply:
probably the symlink created just below which breaks rebuilding without cleaning